sycl/include/sycl/accessor.hpp

```diff
@@ -1835,6 +1844,14 @@ class __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
   bool operator==(const accessor &Rhs) const { return impl == Rhs.impl; }
   bool operator!=(const accessor &Rhs) const { return !(*this == Rhs); }
 
+  iterator begin() noexcept {
```

Comment:
Shouldn't both of these methods be marked const ?

Reply:
Fixed in d68418552d633a57fc40aa720a1a9c4fc5bd7d0a